README.md

```diff
@@ -1,34 +1,55 @@
+Xpring SDK is a set of libraries and services designed to make interaction with protocols from Xpring easy and intuitive. Xpring SDK works consistently across languages and platforms to create a cohesive development experience, regardless of the language, platform, or environment you are building your app in.
 
-## Architecture
-The development experience is comprised of two components:
-- A set of language-specific client-side libraries
-- A server-side component that handles requests from the client-side libraries and proxies them to an XRP node
+### Libraries
 
-### Client-Side Libraries
-Xpring SDK provides the most commonly used functionality on the XRP Ledger. Functionality and programming paradigms are common across all libraries in Xpring SDK.
+Xpring SDK provides commonly used functionality for the XRP Ledger, InterLedger Protocol (ILP) and PayID. Functionality and programming paradigms are common across all libraries in Xpring SDK.
 
-Xpring SDK provides the following features:
+Xpring SDK provides the following features when used with the XRPL Ledger protocol:
 - Wallet generation and derivation (Seed-based or HD Wallet-based)
 - Address validation
 - Account balance retrieval
 - Sending XRP payments
+- Transaction Status
+- Payment History
+
+Xpring SDK provides the following features when used with InterLedger protocols:
+- Sending payments
+- Checking account balance
 
 Xpring SDK is available in the following languages:
 - Java: [Xpring4j](https://github.com/xpring-eng/xpring4j)
 - Swift: [XpringKit](https://github.com/xpring-eng/xpringkit)
 - JavaScript: [Xpring-JS](https://github.com/xpring-eng/xpring-js)
 
-### Server-Side Component
-The server-side component sends client-side requests to an XRP Node.
+### Remote Node
+
+Xpring SDK connects to a remote [rippled node](https://github.com/ripple/rippled) or [interledger node](https://github.com/xpring-eng/hermes-ilp).
 
-To get developers started right away, Xpring currently provides the server side component as a hosted service, which proxies requests from client side libraries to an XRP Node. Developers can reach the endpoint here: `grpc.xpring.tech:80`
+Xpring recommends users of the SDK run their own nodes. However, Xpring recognizes that users may want to rapidly prototype without running their own infrastructure. As a result, we run the following public nodes which users can connect to:
 
-Xpring is working on building a zero-config way for XRP node users to deploy and use the adapter as an open-source component of [rippled](https://github.com/ripple/rippled). Watch this space!
+
+```
+# Rippled
+## Mainnet
+main.xrp.xpring.io:50051         # Outside of a browser
+https://envoy.main.xrp.xpring.io # Inside of a browser
+
+## Testnet
+test.xrp.xpring.io:50051         # Outside of a browser
+https://envoy.test.xrp.xpring.io # Inside of a browser
+
+# ILP
+## Testnet
+hermes-envoy-test.xpring.io
```

Comment:
Not a blocker on teh PR, but we chose dashes instead of dots (i.e., subdomains) because depending on the cert we use, some browsers don't honor the sub-domains (at least that's the way it used to be). 

That said, we should conform our URL structures...

Reply:
Tracked in https://app.asana.com/0/1143722888069470/1167929016291505